.vscode/settings.json

```diff
@@ -1,8 +1,31 @@
 {
+  "files.exclude": {
```

Comment:
Why have we hidden these files? There is a chance we have to delete the `.lake` dir this week, so hidding it is just going to make that harder.

Reply:
it's better if it's hidden
easier on the eyes

if we need the students to delete the `.lake` directory then we'll just tell them to type `rm -r .lake` in their terminals?

Reply:
in fact, how about i write a script that will remove the `.lake` directory anyway, and do `lake exe cache get`?

so all the students would need to do is type in `sh script.sh`

Reply:
Okay, I guess that works!

Reply:
lmao, or we can just say type in
```terminal
rm -r .lake && lake exe cache get
```
basically one command haha